### PR TITLE
Add private icon next to a private artifact

### PR DESCRIPTION
**Pull request by Mark-Powers**
This shows an icon next to an artifact title when the artifact is private. The only other place this information is displayed is inside the sharing settings.

List view:
image

Detail view:
image

### CODE REVIEW

**Review by super-cooper — APPROVED** [review]

**Comment by super-cooper** [comment]
Does this icon have hovertext? I think that would be helpful.

File: sharing_portal/static/sharing_portal/artifacts.css [hunk 1]
```diff
@@ -247,3 +248,15 @@
...
+
+.cardTitleTitle {
+    flex: 1;
+}
+.cardTitlePrivate {
+    flex: 1;
+    text-align: right;
+}
+.artifactDetailPrivate {
+    font-size: 32px;
```
Comment (diurnalist) [inline sharing_portal/static/sharing_portal/artifacts.css:260]: What does it look like if we don't override this?

**Comment by diurnalist** [comment]
We could also throw the icon into the "stats" component, which is re-used in both the list view and the detail view. That's where the icons are currently, I think it would be a decent place to put this one too.

File: sharing_portal/static/sharing_portal/artifacts.css (hunk 1 above)
Comment (Mark-Powers) [reply sharing_portal/static/sharing_portal/artifacts.css:260]: It's a bit too small. 
image

**Pull request merged**